src/CoreBundle/Resources/views/snippets-cms/StaticView/en/cms_welcome.html.twig

```diff
@@ -1 +1,24 @@
-Welcome!
+<div class="card">
+    <div class="card-body">
+        <div class="jumbotron p-1 bg-white mb-0">
+            <h1 class="display-4">Welcome</h1>
+            <p class="lead">in your Chameleon System open source backend</p>
+            <hr class="my-4">
+            <p>
+                Visit our product website for information about Chameleon System and available extensions.
+            </p>
+            <p>
+                <a href="https://www.chameleon-system.de" class="btn btn-primary" target="_blank"><i class="fas fa-external-link-square-alt"></i> chameleon-system.de</a>
```

Comment:
```suggestion
                <a href="https://www.chameleonsystem.com" class="btn btn-primary" target="_blank"><i class="fas fa-external-link-square-alt"></i> chameleonsystem.com</a>
```

(chameleonsystem.com will be configured to display the page in English directly, so we should link to that domain in the English text)

Reply:
ok